### DISCUSSION TITLE
Questions regarding using Mantine in an enterprise project

### DISCUSSION THREAD

**Discussion by royibernthal** [mantinedev/mantine#930] [Q&A] [answered]
Hi,

Mantine looks like a very interesting framework, and we're considering using it in an enterprise project.

I have a few questions:

1) Are there any big / popular companies that use or support Mantine?

2) How long has Mantine been around?

3) Does the styles API really allow full customization of internal components?
One of the main pain points of ui frameworks is not being able to fully customize them to a custom design's demands.

4) Are there any other considerations that need to be taken into account when starting a new project with Mantine?

**Comment by rtivital** [comment, marked as answer]
> Are there any big / popular companies that use or support Mantine?

Yes.

> How long has Mantine been around?

1 year.

> Does the styles API really allow full customization of internal components?

Yes.

> Are there any other considerations that need to be taken into account when starting a new project with Mantine?

Follow getting started section.

I think you have a wrong attitude. Mantine is just a library (not a lifestyle, not a life-changing decision), would you ask [is-integer](https://www.npmjs.com/package/is-integer) library maintainer if it's used in large companies, or how long it's been around? I guess no – you would just use it if it helps you solve problems or make you more productive. Mantine is not different, so there is no point in treating it in any other way.

**Reply by royibernthal** [reply]
Thanks for the quick response.

A UI framework is more crucial than a small utility library and is far more painful to replace down the line, the whole front end development revolves around it.
I'm not entirely sure why you think it's not a big decision :)

Please don't get the wrong idea, I personally think Mantine looks great.
However, in order to get it through the gates of an enterprise one would need to make a good case - the considerations are often strict and do involve the questions I raised.

Is there anywhere I could see a list of the big / popular companies that use and support Mantine?
